README.md

```
@@ -24,13 +27,14 @@ Star the project 🌟 to get releases notification and help growing the communit
-[![API Tests](https://github.com/intuitem/ciso-assistant-community/actions/workflows/backend-api-tests.yml/badge.svg)](https://github.com/intuitem/ciso-assistant-community/actions/workflows/backend-api-tests.yml)
-[![Functional Tests](https://github.com/intuitem/ciso-assistant-community/actions/workflows/functional-tests.yml/badge.svg?branch=main)](https://github.com/intuitem/ciso-assistant-community/actions/workflows/functional-tests.yml)
-![GitHub Release](https://img.shields.io/github/v/release/intuitem/ciso-assistant-community)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fab-smith%2Fciso-assistant-community.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fab-smith%2Fciso-assistant-community?ref=badge_small)
-<a href="https://intuitem.gitbook.io/ciso-assistant"><img src="https://img.shields.io/static/v1?message=Documentation&logo=gitbook&logoColor=ffffff&label=%20&labelColor=5c5c5c&color=F4E28D"></a>
-<a href="https://ca-api-doc.pages.dev/"><img src="https://img.shields.io/static/v1?message=API&logo=swagger&label=%20"></a>
+![GitHub Release](https://img.shields.io/github/v/release/intuitem/ciso-assistant-community?style=for-the-badge)
+![GitHub contributors](https://img.shields.io/github/contributors-anon/intuitem/ciso-assistant-community?style=for-the-badge&color=%235D4596)
+![GitHub Repo stars](https://img.shields.io/github/stars/intuitem/ciso-assistant-community?style=for-the-badge)
+![GitHub forks](https://img.shields.io/github/forks/intuitem/ciso-assistant-community?style=for-the-badge&color=%235D4596)
+![Discord](https://img.shields.io/discord/1155083727932764190?style=for-the-badge&label=Discord)
+<a href="https://intuitem.gitbook.io/ciso-assistant"><img src="https://img.shields.io/static/v1?message=Documentation&logo=gitbook&logoColor=ffffff&label=%20&labelColor=5c5c5c&color=F4E28D&style=for-the-badge"></a>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)